test/unit/filters/CSFilterTest.cpp

```diff
@@ -62,5 +62,7 @@ TEST(CSFilterTest, emptyView)
     filter->prepare(table);
 
     PointViewSet s = filter->execute(table);
-    EXPECT_EQ(s.size(), 0u);
+    EXPECT_EQ(s.size(), 1u);
```

Comment:
Wondering what changed here.

Reply:
Simply that I add the input PointView to the output PointViewSet via list initialization in each of the three filters now, prior to checking whether or not the PointView has any points. I had previously returned with an empty PointViewSet in that case, but now return the PointViewSet, which just happens to contain an empty PointView. It could be reverted if it makes more sense the original way.

Reply:
Nope. That's fine. Thanks for the explanation. I wouldn't have questioned if not for the change.